arbnode/inbox_reader.go

```diff
@@ -512,14 +562,27 @@ func (r *InboxReader) run(ctx context.Context, hadError bool) error {
 					} else if haveAcc != beforeAcc {
 						reorgingDelayed = true
 					}
+					havePrevAcc = haveAcc
 				}
+				log.Trace(
```

Comment:
Unlike the others, this one should be log.Trace because it does unnecessary work.
Another option: separate into log.Debug with most info and log.Trace for the lazyHash. Think if such a solution will help debugging in when setting log-level to debug (vs. the other logs already available). If not - keep all in trace.

Reply:
It's just two hashes so I think it's reasonable to leave it enabled for debug, and it's also not on the block processing thread, but if you think it might be too expensive I can move it to a separate trace log